typings/restify-validator/restify-validator.d.ts

```diff
@@ -0,0 +1,24 @@
+// Type definitions for adding validator to restify
+
+/// <reference path="../express-validator/express-validator.d.ts" />
+/// <reference path="../restify/restify.d.ts" />
+
+declare module "restify" {
+    import http = require('http');
+    import validator = require('express-validator');
+
+    interface Request extends http.ServerRequest {
+        checkBody(field:string, message:string): validator.Validator;
+        checkParams(field:string, message:string): validator.Validator;
+        checkQuery(field:string, message:string): validator.Validator;
+        checkHeader(field:string, message:string): validator.Validator;
+        sanitize(field:string): validator.Sanitizer;
+        onValidationError(func:(msg:string) => void): void;
+        validationErrors(mapped?:boolean) : any;
+    }
+}
+
+declare module "express-validator" {
```

Comment:
Yes, because validator() returns an express.RequestHandler and we need to instead treat it as a restify.RequestHandler.